crawl-ref/source/main.cc

```diff
@@ -3160,7 +3160,8 @@ static void _move_player(coord_def move)
                 mpr("You deftly slip free of the water engulfing you.");
             else //Unless you're a natural swimmer, this takes longer than normal
             {
-                mpr("With effort, you pull free of the water engulfing you.");
+                mpr("It takes you longer to move as you pull free of the water "
```

Comment:
Perhaps "You slowly pull free ..." instead of "With effort, you pull free ..."?

Reply:
Yes that would also work.